karma.ci.conf.js

```diff
@@ -75,10 +75,6 @@ module.exports = function (config) {
     sauceLabs: {
       testName: 'Recurly.js tests',
       recordVideo: true,
-      connectOptions: {
-        port: 5757,
-        logfile: 'sauce_connect.log'
-      },
```

Comment:
Removed this per https://github.com/karma-runner/karma-sauce-launcher/issues/145#issuecomment-335410867